context/JobContext.js

```diff
@@ -0,0 +1,90 @@
+import React, { createContext, useState, useContext } from "react";
+
+// Create a context
+const JobContext = createContext();
+
+// Create a provider component
+export const JobProvider = ({ children }) => {
+  const JobsData = [
+    {
+      jobid: 1,
+      country: "INDIA",
+      city: "PUNE",
+      title: "Application Developer",
+      role: "Software Engineering - Project Role",
+      description: "Application Developer Project Role...",
+      businessArea: "Technology",
+      experience: "5-10",
+      posted: "Posted within last 24 hours",
+      skills: ["Java", "Spring Boot", "Microservices"],
+    },
+    {
+      jobid: 2,
+      country: "INDIA",
+      city: "BANGALORE",
+      title: "Full Stack Developer",
+      role: "Software Engineering - Project Role",
+      description: "Full Stack Development...",
+      businessArea: "Technology",
+      experience: "3-5",
+      posted: "Posted within last week",
```

Comment:
we have to remove this.. as we will fetching data from backend..

Reply:
As of now, we're using hard coded data but soon it will be replaced